packages/core/useStateTransition/index.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import 'vue-tsx-support/enable-check'
+import Vue from 'vue'
+import { storiesOf } from '@storybook/vue'
+import { createComponent, ref } from '../../api'
+import { ShowDocs } from '../../_docs/showdocs'
+import { useStateTransition } from '.'
+
+type Inject = {
+  baseNumber: number
+  number: number
+}
+
+const rand = () => Math.floor(Math.random() * (1000 - 0) + 0)
+
+const Demo = createComponent({
+  setup() {
+    const baseNumber = ref(rand())
+
+    const number = useStateTransition(baseNumber, {
+      duration: 1000,
+      transition: 'easeInOutExpo',
+    })
+
+    return {
+      baseNumber,
+      number,
+    }
+  },
+
+  render(this: Vue & Inject) {
+    // @ts-ignore
+    const Docs = <ShowDocs md={require('./index.md')} />
+
+    const onClick = () => {
+      this.baseNumber = rand()
+    }
+
+    return (
+      <div>
+        <div id='demo'>
+          <button onClick={onClick}>
+            Change number
+          </button>
+          <p>Base number: <b>{this.baseNumber}</b></p>
+          <p>Transitioned number: <b>{Math.round(this.number)}</b></p>
+        </div>
+        {Docs}
+      </div>
+    )
+  },
+})
+
+storiesOf('State', module)
+  .add('useStateTransition', () => Demo as any)
```

Comment:
Maybe simply `useTransition` under `Animation` category would be more straightforward. What do you think?

Reply:
I like that better, I'll go ahead and make the. change 👍